e2e/steps/configurations.go

```diff
@@ -0,0 +1,60 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package steps
+
+import (
+	"path"
+
+	"github.com/elastic/e2e-testing/cli/shell"
+	"github.com/elastic/e2e-testing/e2e"
+	log "github.com/sirupsen/logrus"
+)
+
+// FetchBeatConfiguration where 'configFileName' represents the configuration file, including extension.
+// It will read environment to determine if it must read a file from the local file system, or in the
+// contrary, download a file from Github. In the latter case, it will use a commit of the maintenance branch
+// used in the tests
+func FetchBeatConfiguration(xpack bool, beat string, configFileName string) (string, error) {
```

Comment:
We should probably extract code for configurations management to here, instead of the huge utils.go file :/